scripts/ios/fv/gh-actions-build-with-tests-fv.sh

```diff
@@ -17,12 +17,18 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
+mkdir -p build && cd build
+sudo xcode-select -p
+# Using Xcode_11 pre-installed on node. Otherwise, compilation may fail.
+sudo xcode-select -s /Applications/Xcode_11.2.1.app
+sudo xcode-select -p
 
-mkdir -p build
-cd build
-cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE \
-    -OLP_SDK_BUILD_EXAMPLES=ON \
-    -DBUILD_SHARED_LIBS=ON \
-    ..
-make -j
-cd ..
+cmake ../ -GXcode \
+    -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/iOS.cmake \
+    -DCMAKE_BUILD_TYPE=$BUILD_TYPE \
+    -DPLATFORM=iphoneos \
+    -DOLP_SDK_ENABLE_TESTING=ON \
+    -DSIMULATOR=YES \
+    -DOLP_SDK_BUILD_EXAMPLES=ON
```

Comment:
Why is this needed? Examples we only need to test ones.

Reply:
thanks for finding it, fixed.